src/test/java/seedu/duke/atomichabit/AtomicHabitTest.java

```diff
@@ -0,0 +1,17 @@
+package seedu.duke.atomichabit;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.atomichabit.atomichabit.AtomicHabitManager;
+
+
+public class AtomicHabitTest {
+
+    /**
+     * Will fill with tests after discussion
+     */
+    @Test
+    public void addhabit_correct_output() {
+
+    }
+
+}
```

Comment:
Add newline at the end of file

Reply:
Resolved in 3371f317f0f77df8dbf6210446555e6af3ebac85.